.gitignore

```diff
@@ -29,3 +29,4 @@ MYMETA.*
 .carton/
 /local/
 /log/
+cpanfile.snapshot
```

Comment:
たしかにfork元のリポジトリにもsnapshot入ってないんですけど、これはignoreしなくて良い気がします！

> Make sure you add cpanfile and cpanfile.snapshot to your version controlled repository and commit changes as you update dependencies. This will ensure that other developers on your app, as well as your deployment environment, use exactly the same versions of the modules you just installed.

（https://metacpan.org/pod/Carton より）

とあるように、一緒に開発する人にもバージョンまで全く同じパッケージを入れてもらって、バージョン間の違いに惑わされたりしないようにしたいです。